scalarm_client_csharp/Client.cs

```diff
@@ -422,9 +422,8 @@ private ExperimentStatistics HandleExperimentsStatisticsResponse(IRestResponse<E
 
 		public IList<ValuesMap> GetExperimentResults(string experimentId)
```

Comment:
@Andriu01 - ``error_description`` powinno być opcjonalne. Domyślnie powinno być na false. Trzeba jakoś mądrze nazwać to pole - może "fetchFailed"? I do tego kawałek dokumentacji nad metodą (jakimś tam XML-em - sam wiesz lepiej jak się pisze w C# :) ).